src/Types/RawBytes.purs

```diff
@@ -0,0 +1,79 @@
+-- | A wrapper over `ByteArray` to indicate a byte array with no further specified meaning
+
+module Types.RawBytes
+  ( RawBytes(..)
+  , rawBytesToByteArray
+  , rawBytesFromByteArray
+  , rawBytesToIntArray
+  , rawBytesFromIntArray
+  , rawBytesFromIntArrayUnsafe
+  , rawBytesToHex
+  , byteLength
+  , hexToRawBytes
+  , hexToRawBytesUnsafe
+  ) where
+
+import Data.Newtype (class Newtype, wrap, unwrap)
+
+import Types.ByteArray (ByteArray)
+import Types.ByteArray as BA
+import Data.Maybe (Maybe)
+import Prelude
+import Aeson (class DecodeAeson, decodeAesonViaJson)
+import Data.Argonaut (class DecodeJson, decodeJson)
+import Test.QuickCheck.Arbitrary (class Arbitrary, arbitrary)
+
+-- | An array of Bytes with no information about the content format
+newtype RawBytes = RawBytes ByteArray
+
+instance Show RawBytes where
+  show rb = "(rawBytesFromIntArrayUnsafe" <> show (rawBytesToIntArray rb) <> ")"
```

Comment:
Again, a space after the initial string. And use the hex representation to `rawBytesToHex` show.